hdfs-scheduler/src/main/java/org/apache/mesos/hdfs/scheduler/Reconciler.java

```diff
@@ -117,4 +124,42 @@ private void explicitlyReconcileTasks(SchedulerDriver driver) {
 
     driver.reconcileTasks(tasks);
   }
+
+  private class ReconcileThread extends Thread {
+    private static final int MILLIS_FROM_SECONDS = 1000;
```

Comment:
Milis_from_seconds should be put in the HdfsConstants file especially since I believe it is used elsewhere. We shouldn't copy constants that are being used multiple times